pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -290,4 +292,11 @@ private void submitSegmentGenTask(File localTempDir, URI inputFileURI, int seqId
       }
     });
   }
+
+  private File createLocalInputDateFile(URI inputFileURI, File localInputTempDir) {
+    String inputFileURIPath = inputFileURI.getPath();
+    String fileNameFriendlyPath = UUID.randomUUID().toString();
+    File localInputFileDir = new File(localInputTempDir, fileNameFriendlyPath);
```

Comment:
(minor) shall we rename the variable name since it doesn't have relation to file name? Maybe inline it?
```suggestion
    File localInputFileDir = new File(localInputTempDir, UUID.randomUUID().toString());
```

Reply:
We need `localInputFileDir` to be a variable as it gets deleted in the `finally {}` block I think?